docs/editors/settings.md

```diff
@@ -0,0 +1,138 @@
+# Settings
```

Comment:
Should this be `reference/server.md` instead? It'd be nice to keep the reference docs together.